templates/org.jenkins-ci.slave.jnlp.plist.epp

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-/Apple/DTD PLIST 1.0/EN" "http:/www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <% if $jenkins::slave::quoted_ui_pass { -%>
+  <key>EnvironmentVariables</key>
+  <dict>
+  	<key>JENKINS_PASSWORD</key>
+  	<string><%= $jenkins::slave::quoted_ui_pass -%></string>
+  </dict>
+  <% } -%>
+  <key>Label</key>
+  <string>org.jenkins-ci.slave.jnlp</string>
+  <key>ProgramArguments</key>
+  <array>
+    <string>/usr/bin/java</string>
+    <% if $jenkins::slave::_real_java_args { -%>
+    <string><%= jenkins::slave::_real_java_args -%></string>
```

Comment:
Missing a `$` here. Applied in https://github.com/voxpupuli/puppet-jenkins/pull/1057